scripts/js/footer.js

```diff
@@ -360,37 +360,31 @@ function updateSystemInfo() {
         $("#sysinfo-memory-swap").text("No swap space available");
       }
 
-      color = system.cpu.load.percent[0] > 100 ? "text-red" : "text-green-light";
+      color = system.cpu.load.raw[0] > system.cpu.nprocs ? "text-red" : "text-green-light";
       $("#cpu").html(
         '<i class="fa fa-fw fa-microchip ' +
           color +
-          '"></i>&nbsp;&nbsp;CPU:&nbsp;' +
-          system.cpu.load.percent[0].toFixed(1) +
-          "&thinsp;%"
+          '"></i>&nbsp;&nbsp;Load:&nbsp;' +
+          system.cpu.load.raw[0].toFixed(2) +
+          "&nbsp;/&nbsp;" +
+          system.cpu.load.raw[1].toFixed(2) +
+          "&nbsp;/&nbsp;" +
+          system.cpu.load.raw[2].toFixed(2)
       );
       $("#cpu").prop(
         "title",
-        "Load: " +
-          system.cpu.load.raw[0].toFixed(2) +
-          " " +
-          system.cpu.load.raw[1].toFixed(2) +
-          " " +
-          system.cpu.load.raw[2].toFixed(2) +
-          " on " +
+        "Shown load metrics are load averages for the past 1, 5, and 15 minutes on a system having " +
```

Comment:
We can simplify the text here:

```suggestion
        "Load averages for the past 1, 5, and 15 minutes on a system with " +
```
![image](https://github.com/user-attachments/assets/7d46649c-e274-48b8-a0bd-ae8611ca4698)


Or using a line break:
```suggestion
        "Load averages for the past 1, 5, and 15 minutes\non a system with " +
```
![image](https://github.com/user-attachments/assets/b3c78891-f3c1-4b82-8c01-82aa3fe388ab)